common/config/rush/browser-approved-packages.json

```diff
@@ -1,5 +1,14 @@
 // DO NOT ADD COMMENTS IN THIS FILE.  They will be lost when the Rush tool resaves it.
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/approved-packages.schema.json",
-  "packages": []
+  "packages": [
+    {
+      "name": "handlebars",
+      "allowedCategories": [ "tests" ]
+    },
+    {
+      "name": "handlebars-loader",
+      "allowedCategories": [ "tests" ]
+    }
```

Comment:
This (and maybe `"handlebars"` as well) should go in `nonbrowser-approved-packages.json`

Reply:
Do we need `handlebars` at all?  That seems maybe too advanced for a tutorial project.

@iclanton In the Playground project you used it like this:
```html
<!DOCTYPE html>
<html lang="en">
  <head>
  {{#each scriptsToInclude}}
    <script type="text/javascript" src="{{url}}"></script>
  {{/each}}
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#000000">
    <title>TSDoc Playground</title>
  </head>
```

But doesn't `HtmlWebpackPlugin` already provide its own feature for injecting scripts?

Reply:
I will be on vacation the next 5 days, so to help with the merge, please feel free to make any modifications you wish to this pull request.   I'm okay with anything.

Reply:
Cool. Thanks @jasonswearingen for making this PR!